### PR TITLE
Enabling FT for OSS bert & roberta family models

### PR DESCRIPTION
**Pull request by Anubha98**
Tested out bert & roberta family including 10 models
1. Run success
2. Deployment success
3. Able to infer
Note: all the defaults will be same as HF ones except the finetuning tasks will only be having text-class and Q&A, excluding token-classification for now till we are unblocked.

Experiment: https://ml.azure.com/experiments/id/db8f256e-9fbc-4070-9a93-3f9131bc5a06?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourceGroups/training_rg/providers/Microsoft.MachineLearningServices/workspaces/train-finetune-dev-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

### CODE REVIEW

File: assets/models/system/bert-base-cased/spec.yaml [hunk 1]
```diff
@@ -6,11 +6,15 @@ properties:
   SHA: 5532cc56f74641d4bb33641f5c76a55d11f846e0
   evaluation-min-sku-spec: 4|0|28|56
   evaluation-recommended-sku: Standard_DS4_v2, Standard_D8a_v4, Standard_D8as_v4, Standard_DS5_v2, Standard_DS12_v2, Standard_D16a_v4, Standard_D16as_v4, Standard_D32a_v4, Standard_D32as_v4, Standard_D48a_v4, Standard_D48as_v4, Standard_D64a_v4, Standard_D64as_v4, Standard_D96a_v4, Standard_D96as_v4, Standard_FX4mds, Standard_FX12mds, Standard_F16s_v2, Standard_F32s_v2, Standard_F48s_v2, Standard_F64s_v2, Standard_F72s_v2, Standard_FX24mds, Standard_FX36mds, Standard_FX48mds, Standard_E4s_v3, Standard_E8s_v3, Standard_E16s_v3, Standard_E32s_v3, Standard_E48s_v3, Standard_E64s_v3, Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3, Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4, Standard_ND40rs_v2
+  finetune-min-sku-spec: 4|1|28|176
+  finetune-recommended-sku: Standard_NC12s_v3
```
Comment (novaturient95) [inline assets/models/system/bert-base-cased/spec.yaml:10]: finetune-recommended-sku has to exactly match with allowlisted skus. Please check inference and evaluation example

Comment (Anubha98) [reply assets/models/system/bert-base-cased/spec.yaml]: Thanks @novaturient95 , fixed.

File: assets/models/system/roberta-large-mnli/spec.yaml [hunk 2]
```diff
@@ -111,4 +115,20 @@ tags:
...
+      Standard_NC24s_v3,
+      Standard_NC24rs_v3,
+      Standard_NC4as_T4_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC64as_T4_v3,
+      Standard_ND40rs_v2,
+      Standard_ND96asr_v4
+    ]
 version: 12
```
Comment (BhavanaTumma) [inline assets/models/system/roberta-large-mnli/spec.yaml:134]: Could you please change the versions of the models?

Comment (Anubha98) [reply assets/models/system/roberta-large-mnli/spec.yaml]: Updated

**Review by BhavanaTumma — APPROVED** [review]
LGTM.

**Pull request merged**